File: Classes/TiCoremlRealtimeRecognitionView.m
/**
 * Appcelerator Titanium Mobile
 * Copyright (c) 2009-2017 by Appcelerator, Inc. All Rights Reserved.
 * Licensed under the terms of the Apache Public License
 * Please see the LICENSE included with this distribution for details.
 */

#import "TiCoremlRealtimeRecognitionView.h"
#import "TiCoremlRealtimeRecognitionViewProxy.h"

@implementation TiCoremlRealtimeRecognitionView

- (void)frameSizeChanged:(CGRect)frame bounds:(CGRect)bounds
{
    [(TiCoremlRealtimeRecognitionViewProxy *)[self proxy] adjustFrame];
}

@end

File: Classes/TiCoremlRealtimeRecognitionView.swift
//
//  TiCoremlRealtimeRecognitionView.swift
//  TiCoreml
//
//  Created by Hans Knöchel on 25.02.24.
//

import CoreVideo
import TitaniumKit
import UIKit

@objc(TiCoremlRealtimeRecognitionView)
class TiCoremlRealtimeRecognitionView: TiUIView {
  
  override func frameSizeChanged(_ frame: CGRect, bounds: CGRect) {
    super.frameSizeChanged(frame, bounds: bounds)
    
    if let proxy = proxy as? TiCoremlRealtimeRecognitionViewProxy {
      proxy.adjustFrame()
    }
  }
}
